Source/Scene/Model.js

```diff
@@ -4094,20 +4018,32 @@ define([
                 }
 
                 this._loadResources = new LoadResources();
-                parse(this);
             }
+            parse(this);
         }
 
         var loadResources = this._loadResources;
         var incrementallyLoadTextures = this._incrementallyLoadTextures;
         var justLoaded = false;
 
         if (this._state === ModelState.LOADING) {
-            // Create WebGL resources as buffers/shaders/textures are downloaded
-            createResources(this, frameState);
-
             // Transition from LOADING -> LOADED once resources are downloaded and created.
             // Textures may continue to stream in while in the LOADED state.
+            if (loadResources.pendingBufferLoads === 0 && loadResources.pendingShaderLoads === 0) {
+                if (!this._updatedGltfVersion) {
+                    updateVersion(this.gltf);
```

Comment:
Also, is the `_updatedGltfVersion` flag needed?  Couldn't the version just be upgraded as it was before (note that it was awkwardly implicitly in the `addDefaults` function)?